### PR TITLE
upd: added ability to pull extended tweets

### PR DESCRIPTION
**Pull request by pcgc**

### CODE REVIEW

File: src/TwitterFeed.php [hunk 1]
```diff
@@ -77,6 +78,7 @@ public function __construct(
         $this->userId = $userId;
         $this->excludeReplies = $excludeReplies;
         $this->includeRts = $includeRts;
+        $this->extended = $extended;
```
Comment (ambroisemaupate) [inline src/TwitterFeed.php:81]: @pcgc Be careful, `$extended` var is not defined in your constructor. You can define as constructor optional argument.

File: src/TwitterFeed.php [hunk 2]
```diff
@@ -94,6 +96,7 @@ protected function getFeed($count = 5)
                 "count" => $count,
                 "exclude_replies" => $this->excludeReplies,
                 'include_rts' => $this->includeRts,
+                'tweet_mode' =>  ($this->extended ? 'extended' : '')
```
Comment (ambroisemaupate) [inline src/TwitterFeed.php:99]: @pcgc Does this option could be useful for TwitterSearchFeed too?

**Review by ambroisemaupate — CHANGES_REQUESTED** [review]
Thanks @pcgc for your PR and your work. Here are some comments before merging.

File: src/TwitterFeed.php (hunk 1 above)
Comment (pcgc) [reply src/TwitterFeed.php:81]: line 67 
`$extended = false`

Comment (ambroisemaupate) [reply src/TwitterFeed.php:81]: My bad

File: src/TwitterFeed.php (hunk 2 above)
Comment (pcgc) [reply src/TwitterFeed.php:99]: Twitter search api doesn't offer this

https://developer.twitter.com/en/docs/tweets/search/api-reference/get-search-tweets

Comment (nikossvnk) [reply src/TwitterFeed.php:99]: this is now supported by search api too. I opened a pull request

**Pull request merged**